packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.tsx

```diff
@@ -0,0 +1,187 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+/* tslint:enable:no-unused-variable */
+import * as PropTypes from 'prop-types';
+import {
+  autobind,
+  BaseComponent,
+  css,
+  findScrollableParent
+} from '../../Utilities';
+import { IScrollablePaneProps } from './ScrollablePane.Props';
+import { Sticky } from '../../Sticky';
+import * as stylesImport from './ScrollablePane.scss';
+const styles: any = stylesImport;
+
+export class ScrollablePane extends BaseComponent<IScrollablePaneProps, {}> {
+  public static childContextTypes = {
+    subscribe: PropTypes.func,
+    addStickyHeader: PropTypes.func,
+    removeStickyHeader: PropTypes.func,
+    addStickyFooter: PropTypes.func,
+    removeStickyFooter: PropTypes.func
```

Comment:
Collapse all of these into a single `scrollablePane` object with a defined interface.